about.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <link rel="stylesheet" href="about.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css"
+      integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato&display=swap"
+      rel="stylesheet"
+    />
+    <script src="index.js" defer></script>
+    <title>About</title>
+  </head>
+  <body>
+    <header class="header">
+      <ul class="header-list1">
+        <li class="listA listA-1">
+          <i class="fa-brands fa-facebook-f"></i>
+        </li>
+        <li class="listA listA-2"><i class="fa-brands fa-twitter"></i></li>
+        <li class="listA listA-3">English</li>
+        <li class="listA listA-4">My Page</li>
+        <li class="listA listA-5">Layout</li>
+      </ul>
+
+      <nav class="navbar">
+        <a href="./index.html"
+          ><img src="./images/WFP_logo.png" alt="" class="logo"
+        /></a>
+
+        <i class="fa-solid fa-bars hambuger"></i>
+
+        <ul class="navbar-list1 visibility">
+          <li><a href="./about.html">About</a></li>
+          <li class="menu">Program</li>
+          <li class="menu">Join</li>
+          <li class="menu">Sponsor</li>
+          <li class="menu">News</li>
+          <li class="campagne">CC Campagne</li>
+        </ul>
+      </nav>
+    </header>
+
+    <section class="about">
+      <div class="aboutcontainer">
+        <h1 class="h1">ENVIRONMENT PROTECTION GLOBAL SUMIT 2015</h1>
+
+        <div class="topic1container">
+          <p class="topic1">
+            The Global Summit brings together the community of experts,
+            academics, and activists who comprise the Creative Commons affiliate
+            network in a different country every two years. This year, we hope
+            to expand our invitation list including organizations and
+            individuals who want to work with us on shared projects that advance
+            the cause of the Commons, free culture and open knowledge. So if
+            you're active and engaged in the worlds of open content and
+            knowledge - free software advocates, wikipedians, Open knowledge,
+            galleries, libraries, museums, archives, governments and
+            foundations, lawyers, and activists - we hope you'll consider
+            joining us this year to build a stronger, more vibrant commons
+            together
+          </p>
```

Comment:
- [ ] Please ensure that your design matches the original design as much as possible. Ensure the width flexibility of the paragraph in all the desktop screen sizes. If you notice carefully, you can see that the texts are overlapping in the articles section. Please ensure to fix this to make sure that your design looks professional and has greater UI ☺️.

Your design at about 785px
![image](https://github.com/enocol/capstone/assets/99394374/02554b38-ae96-4909-b7c7-c08246dcc2f9)

original design
![image](https://github.com/enocol/capstone/assets/99394374/cf3e1b35-b3ed-4d5c-baae-1f602aa68628)